src/pages/data-privacy.tsx

```diff
@@ -5,18 +5,34 @@ import { Layout } from '../components/layout/layout';
 import { ContentBlockContainer } from '../components/layout/content-block-container';
 import { ContentfulRichText } from '../components/content/rich-text/rich-text';
 import { ContentfulPage, ContentfulRichTextType } from '../types';
+import { Button } from '../components/ui/buttons/button';
+import { useTranslation } from 'react-i18next';
+import Callout from '../components/ui/callout/callout';
+import styled from 'styled-components';
 
 interface DataPrivacyPageQueryProps {
   contentfulPage: ContentfulPage;
   locales: LocalesQueryProps;
 }
 
+const StyledText = styled.p`
+  margin: 0;
+  margin-bottom: 1rem;
```

Comment:
Is there a specific reason for using `rem` here? If not, I would use `px` instead. This way, we are consistent with the existing code and the margin is always the same as the top and bottom padding of the `Callout`, not dependent on the root font size. 

```suggestion
  margin-bottom: 16px;
```